src/servers/src/mysql/handler.rs

```diff
@@ -221,6 +221,10 @@ impl MysqlInstanceShim {
 impl<W: AsyncWrite + Send + Sync + Unpin> AsyncMysqlShim<W> for MysqlInstanceShim {
     type Error = error::Error;
 
+    fn version(&self) -> String {
+        std::env::var("GREPTIMEDB_MYSQL_SERVER_VERSION").unwrap_or_else(|_| "8.4.2".to_string())
```

Comment:
non-blocking: Maybe we should document this env in our docs.

Reply:
Well, this is the "dark side" of greptimedb, let's not create more confusing to our users, just keep it deep inside our codes.